examples/importation.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from kinto_client.importer import KintoImporter
+
+
+class DummyImporter(KintoImporter):
+    collection_permissions = {'read': ["system.Everyone"]}
+    fields = ('name', 'protocol')
```

Comment:
This is the list of fields that are to be compared to decide whether local and records are the same.